adapters/saml/undertow/pom.xml

```diff
@@ -32,12 +32,6 @@
 
     <dependencies>
         <dependency>
-            <groupId>org.jboss.logging</groupId>
```

Comment:
Yes please. You can send a separate PR for that change if you want (no need to create JIRA).